charts/bulk-scan-processor/values.preview.template.yaml

```diff
@@ -19,31 +19,21 @@ java:
       secretRef: storage-secret-{{ .Release.Name }}
       key: accessKey
   environment:
-    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
-    STORAGE_BLOB_PUBLIC_KEY: "nonprod_public_key.der"
     STORAGE_URL: "https://$(STORAGE_ACCOUNT_NAME).blob.core.windows.net"
-    STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
     QUEUE_ENVELOPE_SEND: "$(SB_CONN_STRING);EntityPath=envelopes"
     QUEUE_NOTIFICATIONS_SEND: "$(SB_CONN_STRING);EntityPath=notifications"
     QUEUE_NOTIFICATIONS_READ: "$(SB_CONN_STRING);EntityPath=notifications"
     QUEUE_PROCESSED_ENVELOPES_READ: "$(SB_CONN_STRING);EntityPath=processed-envelopes"
-    SCAN_ENABLED: "true"
-    SCAN_DELAY: "4000"
-    REUPLOAD_ENABLED: "true"
-    REUPLOAD_DELAY: "4000"
-    SPRING_MAIL_HOST: "false"
-    NOTIFICATIONS_TO_ORCHESTRATOR_TASK_ENABLED: "true"
-    NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
```

Comment:
perhaps answers previous questions too: i bet this is deleted because of the defaults in main config?